.github/workflows/CI2.yml

```diff
@@ -310,6 +321,19 @@ jobs:
           cmake -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$HOME/ALL
           cmake --build build --parallel 2
           cmake --install build
+      - name: Checkout Silo
+        if: ${{ matrix.silo == 'Silo' }}
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: LLNL/Silo
+          ref: 4.11
+          path: silo
+      - name: Build Silo
+        if: ${{ matrix.silo == 'Silo' }}
+        working-directory: silo
+        run: |
+          ./configure CC=mpicc CXX=mpicxx --enable-fortran=no --prefix=$HOME/silo
+          make install
```

Comment:
we are really getting to the point, where we should build all deps, but kokkos (and arborX) in the CI-Container.
And then force CMake to not find it using `-DCMAKE_DISABLE_FIND_PACKAGE_<Foo>=ON`.

Reply:
Added - I checked that the non-Silo builds do not include Silo